src/bp/ui-shared/src/blueprint.css

```diff
@@ -573,6 +573,11 @@ a > .bp3-code {
   font-size: 12px;
   min-width: 30px;
   min-height: 30px;
+  color: #182026;
+  transition: 0.3s;
```

Comment:
does this work? I thought you'd need to specify either a single property or "all" 🤔

Reply:
Yep the transition works. I don't really understand what you mean by this:

> single property or "all"

Reply:
I've always used it with `transition: background 0.3s;` or `transition: all 0.3s;`